src/sender.js

```diff
@@ -32,6 +33,8 @@ const nonceLock = `lock:${config.id}:nonce`
 const nonceKey = `${config.id}:nonce`
 let chainId = 0
 
+const maxProcessingTime = process.env.MAX_PROCESSING_TIME
```

Comment:
We need to define a default value which will be used if `MAX_PROCESSING_TIME` was missed by some reason in the configuration file.

Reply:
What risks can you see if we enable the watchdog by default? In this case the user could disable the watchdog by zeroing the configuration parameter explicitly.

Reply:
That is a bit complicated. Javascript treats 0 as a falsy value, so doing `0 || 10` evaluates to `10`. 

On the other hand, the environment variables are strings, so `'0' || 10` does evaluate to `'0'`, but it's not good to rely on that; we could easily "break" it if we add environment variables validation and casting, like we did in `poa-bridge-contracts/deploy`.

A possible solution is to use a negative value to disable it. That will work both with strings and with numbers: `'-1' < 0` is equivalent to `-1 < 0`.

Reply:
I do not understand this.
Could you use something like:
```
if (process.env.MAX_PROCESSING_TIME === '0') {
...
} else {
...
}
```

And later, when you _add environment variables validation and casting_ you will change this.

Reply:
That can work. This is what I did (see f38688c869be9fcd5e8dceb36b31637769a6f802):
1. If value is explicitly set to 0, the watchdog is not used.
2. If the value is not set, it is computed using 4 * maxPollingTime
3. Otherwise, the set value is used

I didn't want to add this logic twice (in the watcher and the sender), so I moved it to the config. And since the senders weren't including the base config, I also added it to them. 

What do you think?